### PR TITLE
Task 5 - Add Inventory Redux Actions and Reducer

### PR DESCRIPTION
**Pull request by jditch**
See ticket #66 for full specifications

Notes:
The `INVENTORY_CREATE` and `INVENTORY_DELETE` actions will need to be tested when the UI supports their use

### CODE REVIEW

**Comment by jditch** [comment]
The `openSuccess` alert seems to be causing the webpage to freeze after a live recompilation occurs during development (caused by saving the `inventory/index.js` file). This is not expected to be an issue for users, but I am unsure if this is indicative of another problem.

File: ui/src/ducks/inventory/index.js [hunk 1]
```diff
@@ -1,17 +1,73 @@
...
 
-export const findInventory = createAction(actions.INVENTORY_GET_ALL, () => {
-  //TODO
-})
+export const findInventory = createAction(actions.INVENTORY_GET_ALL, () => 
+  (dispatch, getState, config) => axios
+    .get(`${config.restAPIUrl}/inventory`)
+    .then((suc) => {
+      dispatch(refreshInventory(suc.data))
+      dispatch(openSuccess("INVENTORY_GET_ALL Success"))
```
Comment (lndyaouie) [inline ui/src/ducks/inventory/index.js]: Rename all `openSuccess` alerts to a more user-friendly message (e.g., "INVENTORY_GET_ALL Success" could be changed to "Retrieved all inventories", removing the action name with some description).

**Review by lndyaouie — CHANGES_REQUESTED** [review]

**Comment by lndyaouie** [comment]
When saving a file in React, the webpage should automatically refresh to reflect any changes or saved files. If a refresh or hot reload does not occur, a webpage freeze is expected, which would require refreshing the page. This problem is due to the `package.json`'s dependencies (e.g.,  "react-scripts").

Unless the problem persists or I misunderstood the issue, I would ignore the webpage freezing.

File: ui/src/ducks/inventory/index.js (hunk 1 above)
Comment (jditch) [reply ui/src/ducks/inventory/index.js]: Changes made in commit 964580f8b9c107fc99b66c123075ac13383a93eb

**Review by lndyaouie — APPROVED** [review]

**Pull request merged**